__tests__/elements/map/boundary_type_box.test.js

```diff
@@ -0,0 +1,192 @@
+import {VersionController} from "../../../src/js/versions/version_controller.js";
+import {Version} from "../../../src/js/versions/version.js";
+import {Component} from '../../../src/js/utils';
+import Controller from "../../../src/js/controller";
+import {BoundaryTypeBox} from "../../../src/js/map/boundary_type_box.js";
+
+import html from '../../../src/index.html';
+import {Config as SAConfig} from "../../../src/js/configurations/geography_sa";
+
+
+describe('Check Boundary type box options', () => {
+    beforeEach(() => {
+        document.body.innerHTML = html;
+    });
+    let config = new SAConfig();
+
+    let versionGeometries = {
+        "2011 Boundaries": {
+            "children": {
+                "mainplace": {
+                    "type": "FeatureCollection",
+                },
+                "ward": {
+                    "type": "FeatureCollection",
+                }
+            },
+        },
+        "2016 Boundaries": {
+            "children": {
+                "equal area hexagon": {
+                    "type": "FeatureCollection",
+                }
+            }
+        }
+    };
+
+    let selectElement = '.map-geo-select';
+
+    let preferredChildren = {
+        "district" : ["mainplace", "ward", "equal area hexagon"]
+    }
+
+    // version - 2011
+    let version1 = new Version("2011 Boundaries", true);
+    version1.exists = true;
+
+    // verison - 2016
+    let version2 = new Version("2016 Boundaries", true);
+    version2.exists = true;
```

Comment:
I think it is better to move these inside `beforeEach()` function too

Reply:
Done 👍